roles/ceilometer-control/handlers/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: restart ceilometer services
+  service: name={{ item }} state=restarted must_exist=false
+  when: restart|default('True')
+  with_items:
+    - ceilometer-api
+    - ceilometer-collector
+    - ceilometer-agent-notification
+    - ceilometer-agent-central
+    - ceilometer-alarm-evaluator
+    - ceilometer-alarm-notifier
```

Comment:
I just noticed that must_exist=false.... so that's how we avoid that issue. making the change now